src/sentry/options/defaults.py

```diff
@@ -376,7 +376,10 @@
 # removed once it is fully rolled out.
 register("symbolicate-event.low-priority.metrics.submission-rate", default=0.0)
 
+# This is to enable the ingestion of suspect spans by project ids.
 register("performance.suspect-spans-ingestion-projects", default={})
+# This is to enable the ingestion of suspect spans by project groups.
+register("performance.suspect-spans-ingestion.rollout-rate", default=0)
```

Comment:
@Zylphrex would using `FLAG_PRIORITIZE_DISK` here be preferred?

Reply:
Hm... I'll double check 👍